gradle/dependencies.gradle

```diff
@@ -40,9 +40,10 @@ ext.deps = [
                 ]
         ],
         "kotlin": [
-                stdlib: "org.jetbrains.kotlin:kotlin-stdlib:${versions.kotlin}",
-                reflection: "org.jetbrains.kotlin:kotlin-reflect:${versions.kotlin}"
-
+                compiler: "org.jetbrains.kotlin:kotlin-compiler-embeddable:${versions.kotlin}",
```

Comment:
Let's create a task and track about moving off of embeddable

Reply:
Sure - can you open an issue with a description of why moving off of embeddable is preferable?